py/picca/delta_extraction/data.py

```diff
@@ -30,6 +31,7 @@
     "lambda min rest frame": 1040.0,
     "minimum number pixels in forest": 50,
     "rejection log file": "rejection_log.fits.gz",
+    "minimal snr": None,
```

Comment:
I would not add `None` as a value here. Also, it's not trivial to figure out the default value. How about adding `minimal snr bao3d` and `minimal snr pk1d` with the actual defaults here? I would then replace the block in line 209 by
```
if self.analysis_type == "BAO 3D":
    self.min_snr = config.getfloat("minimal snr bao3d")
elif self.analysis_type == "PK 1D":
    self.min_snr = config.getfloat("minimal snr pk1d")
if self.min_snr is None:
    raise  DataError(
                "Missing arguments 'minimal snr bao3d' (if 'analysis type' = 'BAO 3D') or ' minimal snr pk1d' (if 'analysis type' = 'Pk1d') required by Data")
```

Reply:
I'm also missing a `self.min_snr = None` between lines 95 and 96. Variables should always be declared in the `__init__` method

Reply:
hm, it's a bit weird to have two parameters dealing with that, would it be fine having a None default for this and defining 2 defaults that are loaded in the respective case?

Reply:
ok, I think I get why the None is bad... It will screw up the testing environment.

Reply:
changed it according to your suggestion

Reply:
or would you think that having 2 default values defined for a joint config parameter might work? My previous testing issue was actually something else

Reply:
having two default values for a single parameter should work as well, as you wish